src/panoramic/cli/husky/core/model/models.py

```diff
@@ -193,16 +192,6 @@ def identifier_attributes(self) -> Set[ModelAttribute]:
     def identifier_taxon_slugs(self) -> Set[str]:
         return {attr.taxon for attr in self.identifier_attributes}
 
-    @property
-    def physical_data_source(self) -> str:
```

Comment:
In thi case, you also have to change `full_object_name` method, because it starts from index 1 instead of 0